custom-completions/gh/gh-completions.nu

```diff
@@ -568,3 +568,37 @@ export def "gh pr view inlined-comments" [
       | select user.login body diff_hunk
       | rename user comment diff )
 }
+
+def "gh get stars" [
+    end_cursor: string = ""  # endCursor from a previous query
+    --first: int = 100  # returns the first n elements from the list
+] {
+    # https://docs.github.com/en/graphql/reference/objects#user
+    ^gh api graphql -F $'first=($first)' -F $'endCursor=($end_cursor)' -f query='
+      query($first: Int, $endCursor: String!){
+        viewer {
+          starredRepositories(first: $first, after: $endCursor, orderBy: {field: STARRED_AT, direction: DESC}) {
+            edges { node { url description } starredAt }
+            pageInfo { hasNextPage endCursor }
+          }
+        }
+      }
+    '
+    | from json | select data.viewer.starredRepositories.edges data.viewer.starredRepositories.pageInfo
+    | rename stars pageInfo
+}
+
+export def "gh my stars" [] {
+    mut stars = []
+    mut end_cursor = ""
+    loop {
+        let $part = gh get stars $end_cursor
+        $stars = $stars | append $part.stars
+        if $part.pageInfo?.hasNextPage? == true {
+            $end_cursor = $part.pageInfo.endCursor
+        } else {
+            break
+        }
+    }
+    $stars | flatten | update cells --columns [starredAt] {$in| date humanize} | sort-by starredAt
```

Comment:
I think here I would've used `$in | into datetime` instead of `date humanize` because `date humanize` is a string but `into datetime` is a datetime and will be represented as a humanized date by default and could also follow the config section where table dates can be specifically formatted. Lastly, since it's a string, you can't do anything else with the starredAt column based on dates.

Reply:
Aha, ok I assumed I was geting a datetime and that makes me realize that the whole sorting byt date is completely wrong. Addressed in #1110